.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
```

Comment:
Good file, but I would put it in its own PR just for single purpose PRs (just my opinion)